src/components/Auth/UserFormDialog/controls/controls.module.scss

```diff
@@ -24,21 +30,46 @@
 }
 
 .customAttributesWrapper {
-  padding: 0;
-  border: none;
-  margin: 0;
+  :global {
+    label.mdc-text-field--textarea,
+    textarea {
+      height: 66px;
+    }
+  }
+}
+
+.customAttributeWrapper {
   display: flex;
+}
 
-  legend {
-    padding: 0;
-    padding-inline-start: 0;
+.signInWrapper {
+  :global .mdc-list-divider {
+    // Negative margin to take full size
+    margin: 24px -24px;
+
+    &.bottom-divider {
+      margin: -12px -24px;
+    }
   }
+}
+
+.signInHeader {
+  margin-bottom: 20px;
 
-  [role='group'] {
-    display: flex;
+  :global .mdc-typography--body1 {
+    color: $COLOR_TEXT_PRIMARY;
```

Comment:
Please use theming like `<Typography use="body1" theme="textPrimaryOnBackground">` in the JSX file instead, to stay consistent with the rest of the project.